docs/modules/concepts/pages/processing/elements/receiving.adoc

```diff
@@ -0,0 +1,35 @@
+= Email Reception
+:navtitle: Receiving
+
+== Overview
+
+There are various situations in which an email can be "received". The most easily understood
+is mail received from a remote client via an <<SMTP>> session. However, although less visible,
+<<Local,locally received>> email is also an important part of an email system. Although less common and
+not as obvious, email can also be received via <<IMAP>>.
+
+
+
+=== SMTP
+
+The most obvious and easily-understood method that an email server handles "email reception"
+is via an SMTP session. When receiving email, the server is acting in the role of an SMTP server,
+responding to requests by a remote SMTP client.
+
```

Comment:
== LMTP

Reply:
Right! I will add this subsection.

Reply:
Actually, did you mean in your comment that the "Local" section should be entitled "LMTP"? Or did you mean that a separate subsection should be added for LMTP? (I am still having a bit of trouble understand what LMTP even is, so it is difficult for me to write about it.)

Reply:
@chibenwa, I just added the content to the "chaining" section that this link points to. Please let me know if you still think this is problematic.

Reply:
I am reading the spec now to try to understand it better.

Reply:
Updated.